doc/lsst.meas.extensions.scarlet/overview.rst

```diff
@@ -0,0 +1,217 @@
+In other words, every method that we have come up with to generate a better detection image is more expensive computationally than just running the scarlet optimizer for a few extra iterations to converge to the same solution.
+
+Once we have fit the morphology for each component of each source, we use a least-squares algorithm to simultaneously fit the SED of each source *in scarlet model space*.
+Because we normalize the morphology, including the intensity information in the SED, this is different than the observed SED, as the PSF in each band will distribute the model flux differently.
+In this case the computational savings of the least-squares SED fit *is* significant enough to justify its use.
+
+.. note::
+    For *very* large (approx *1k* × *1k* pixel) blends the least squares fit runs into memory issues, since the size of the array to fit is roughly ``number of sources × number of bands × width × height)``.
+    In that case setting :attr:`~lsst.meas.extensions.scarlet.ScarletDeblendConfig.setSpectra` to ``False`` is recommended.
+
+.. note::
+    An interesting fact about optimization in scarlet is that the best initialization (in terms of 𝛘²) will not necessarily result in the fastest runtime or even the most optimal fit.
+    The interplay between the particular gradient descent method used and proximal operators is complicated and often certain irregularities (like sources with slightly worse positions) can result in improved optimization in terms of both runtime and log-likelihood.
+
+.. _optimization:
+
+Optimization
+^^^^^^^^^^^^
+
+In addition to being multi-band, the other main difference between *scarlet* and the single-band deblender is that the *scarlet* models are iteratively optimized using a gradient-descent-like algorithm.
+The extra time to run this optimizer is the primary reason that *scarlet* is not used on the single-visit exposures.
+We have tested a number of different algorithms over the years and so far we have achieved the best performance using proxmial ADAM, as described in `Melchior et al. 2019 <https://arxiv.org/abs/1910.10094>`_.
+
+For example, the blend above can be fit in 11 iterations to get the following results, using the *scarlet* models to re-apportion the flux from the observation in each band:
+
+.. image:: images/scarletModel0.png
+.. image:: images/scarletModel1.png
+.. image:: images/scarletModel2.png
+.. image:: images/scarletModel3.png
+
+While much improved, the models still aren't perfect.
+Notice that the *scarlet* model for source 0 has stray flux to the upper left and lower right that (fortunately) doesn't matter in the flux re-apportioned model, however sources 1 and 2 have their flux slightly truncated due the the presence of their neighbor in both the *scarlet* models and flux re-apportioned models.
```

Comment:
How/why does the stray flux end up like that? I would understand if it were in the direction of another bright source but it seems to go perpendicular to neighbours instead.

Reply:
From investigations in DM-36324 it looks like the truncate PIFF PSFs are wreaking havoc with our deblender models in a way that I did not appreciate. We saw similar effects in the PSF sources, but for much brighter sources. At first we thought it was an issue with the optimizer but now, with more data, it looks like the problem is that PSF isn't large enough, which means there is no feasible monotonic model to capture all of the flux in brighter sources, so the optmizer gets stuck. I'm working on investigating this now and there will probably be an upcoming talk about it.

Reply:
Ok, then again I would note that this seems to be the optimizer's fault, rather than a problem with model selection or anything else the user could try to fix.